src/commonMain/kotlin/fr/acinq/lightning/payment/IncomingPaymentHandler.kt

```diff
@@ -84,8 +87,10 @@ class IncomingPaymentHandler(val nodeParams: NodeParams, val walletParams: Walle
             nodeParams.nodePrivateKey,
             description,
             PaymentRequest.DEFAULT_MIN_FINAL_EXPIRY_DELTA,
-            invoiceFeatures,
+            nodeParams.features,
             randomBytes32(),
+            // We always include a payment metadata in our invoices, which lets us test whether senders support it
+            randomBytes(64).toByteVector(),
```

Comment:
nit:  use `randomBytes64()` ?

Reply:
Updated in https://github.com/ACINQ/lightning-kmp/pull/313/commits/aa81e7d069416777250c9bc709955cf6b8898f60 to use a single byte to avoid making invoices too big